docs/content/concepts/io-management/io-managers.mdx

```diff
@@ -145,14 +145,14 @@ def downstream_asset(upstream_asset):
 defs = Definitions(
     assets=[upstream_asset, downstream_asset],
     resources={
-        "io_manager": ConfigurablePickledObjectS3IOManager(
+        "io_manager": S3PickleIOManager(
             s3_resource=S3Resource(), s3_bucket="my-bucket"
         ),
     },
 )
 ```
 
-This example also constructs an <PyObject module="dagster_aws.s3" object="S3Resource" /> because the <PyObject module="dagster_aws.s3" object="ConfigurablePickledObjectS3IOManager" /> depends on the S3 resource. For more information on how these resource-resource dependencies are modeled, refer to the [resources](/concepts/resources#resources-which-depend-on-other-resources) documentation.
+This example also constructs an <PyObject module="dagster_aws.s3" object="S3Resource" /> because the <PyObject module="dagster_aws.s3" object="S3PickleIOManager" /> depends on the S3 resource. For more information on how these resource-resource dependencies are modeled, refer to the [resources](/concepts/resources#resources-which-depend-on-other-resources) documentation.
```

Comment:
This PyObject link isn't working for me - do we need to re-run Sphinx to get this to show up in the API docs correctly?

Reply:
maybe - i'll investigate

Reply:
ah ok it was really simple. i didn't add the new classes to the API docs in these PRs, instead i added them in this [stacked pr](https://github.com/dagster-io/dagster/pull/15833) since i was already doing a lot of reorganizing of the pages in that PR and rectifying the merge conflicts seems harder than just making the changes there. 

So in that PRs docs preview the PyObjects correctly direct for S3 and ADLS2. But the GCS one isn't working for some reason even though it's added to sphinx correctly (as far as i can tell). So i'm still looking into that one

Reply:
got the second problem figured out and now everything should be clickable in the PR i linked above!

Reply:
Just checked it out - thank you! It looks great!